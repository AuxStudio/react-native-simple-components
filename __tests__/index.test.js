import React from 'react';
import renderer from 'react-test-renderer';

import { Button } from '../';

// We only need to test one of the components for this test
describe('react-native-simple-components', () => {
  it('imports the module', () => {
    expect(renderer.create(<Button />)).toMatchSnapshot();
  });
});
