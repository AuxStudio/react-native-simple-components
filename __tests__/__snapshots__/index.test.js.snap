// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-native-simple-components imports the module 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "#E0E0E0",
      "flexDirection": "row",
      "justifyContent": "center",
      "opacity": 1,
      "padding": 16,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#212121",
          "fontSize": 17,
        },
        undefined,
      ]
    }
  >
    Button
  </Text>
</View>
`;
